examples/type-check-for-meta/custom-typing.ts

```diff
@@ -0,0 +1,16 @@
+type MetaOptions = {
+    device: 'mobile' | 'desktop' | 'tablet';
+    skip: boolean;
+};
+  
+declare module 'testcafe' {
+    global {
+        interface TestFn {
+            meta(options: MetaOptions): this;
+        }
+
+        interface TestFn {
+            meta<T>(options: T): this;
+        }
```

Comment:
Should it work in that way?
```suggestion
        interface TestFn {
            meta(options: MetaOptions): this;
            meta<T>(options: T): this;
        }
```